pkg/controllers/management/node/controller.go

```diff
@@ -29,6 +32,7 @@ import (
 const (
 	defaultEngineInstallURL = "https://releases.rancher.com/install-docker/17.03.2.sh"
 	amazonec2               = "amazonec2"
+	credentialNS            = "cattle-global-data"
```

Comment:
I feel like we already use this namespace name in several places; can you define the global ns in some common util file, and ref from there in all the places